src/mh-core.jl

```diff
@@ -196,18 +196,18 @@ function AbstractMCMC.step!(
     model::DensityModel,
     spl::MetropolisHastings,
     ::Integer,
-    params_prev::Transition;
+    params_prev;
     kwargs...
 )
     # Generate a new proposal.
     params = propose(rng, spl, model, params_prev)
 
     # Calculate the log acceptance probability.
-    logα = logdensity(model, params) - logdensity(model, params_prev) + 
-        q(spl, params_prev, params) - q(spl, params, params_prev)
+    logα = logdensity(model, params) - logdensity(model, params_prev) 
+        + q(spl, params_prev, params) - q(spl, params, params_prev)
 
     # Decide whether to return the previous params or the new one.
-    if -Random.randexp(rng) < logα
+    if log(rand(rng)) < logα
```

Comment:
IMO
```suggestion
    if -Random.randexp(rng) < logα
```
should be better distributed since `rand()` is limited by `eps()` whereas `randexp()` is not.